### PR TITLE
remove deprecated pickImage method

### PR DESCRIPTION
**Pull request by guyo13**

### CODE REVIEW

File: lib/components/contribute_photo_dialog.dart [hunk 1]
```diff
@@ -31,6 +31,7 @@ class _ContributePhotoDialogState extends State<ContributePhotoDialog> {
   File _image;
   String _uploadToken;
   bool _isUploading = false;
+  ImagePicker imagePicker = ImagePicker();
```
Comment (jfschmakeit) [inline lib/components/contribute_photo_dialog.dart]:
```suggestion
  final _imagePicker = ImagePicker();
```
Per the README for the image_picker package, it looks like this should be final.

File: lib/components/contribute_photo_dialog.dart [hunk 2]
```diff
@@ -132,10 +133,9 @@ class _ContributePhotoDialogState extends State<ContributePhotoDialog> {
   Future _getImage(BuildContext context) async {
     // Use the image_picker package to prompt the user for a photo from their
     // device.
-    final File image = await ImagePicker.pickImage(
+    final File image = await imagePicker.getImage(
```
Comment (jfschmakeit) [inline lib/components/contribute_photo_dialog.dart]:
```suggestion
    final File image = await _imagePicker.getImage(
```

**Review by jfschmakeit — CHANGES_REQUESTED** [review]
Thanks for submitting this pull request!

We also need to bump the version of the `image_picker` dependency in the `pubspec.yaml` file.

**Comment by guyo13** [comment]
@jfschmakeit I added the changes you requested

**Comment by jfschmakeit** [comment]
Thanks the code looks good now!

Please also update the `image_picker` dependency to version `^0.6.7+4` in the `pubspec.yaml` file.  (The new methods were only added in `0.6.7` so we need to update the dependency too.)

Once this is merged, I'll also update the steps in the codelab itself to use this new code.

**Comment by guyo13** [comment]
Done !

**Comment by jfschmakeit** [comment]
Apologies for the delay - but thanks for preparing this change. I will go ahead and update the steps in the codelab as well.

**Pull request merged**